scripts/globals/log_ids.lua

```diff
@@ -219,3 +219,34 @@ ROV =
     ['full_name'] = "Rhapsodies of Vana'diel",
     ['mission_log']= 13
 };
+
+QUEST_LOGS = {
+    [0] = "SANDORIA",
+    [1] = "BASTOK",
+    [2] = "WINDURST",
+    [3] = "JEUNO",
+    [4] = "OTHER_AREAS",
+    [5] = "OUTLANDS",
+    [6] = "TOAU",
+    [7] = "WOTG",
+    [8] = "ABYSSEA",
+    [9] = "SOA",
+    [10] = "COALITION",
+};
+
+MISSION_LOGS = {
```

Comment:
I totally missed this before I hit the biutton, my bad. But I am pretty sure these values was already defined inside the other tables.